docs/Quickstart.md

```diff
@@ -12,16 +12,16 @@ This guide assumes you have completed the steps to create a new Rails app using
 
 Your local app needs to be accessible from the public Internet in order to install it on a Shopify store, to use the [App Proxy Controller](/lib/generators/shopify_app/app_proxy_controller/templates/app_proxy_controller.rb) or receive [webhooks](/docs/shopify_app/webhooks.md).
 
-Use a tunneling service like [Cloudflare](https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/run-tunnel/trycloudflare/) to make your development environment accessible to the internet.
+Follow the [*Embed the app in Shopify*](https://shopify.dev/tutorials/build-rails-react-app-that-uses-app-bridge-authentication#step-3-embed-the-app-in-shopify) section of [*Build a Shopify app with Rails, React, and App Bridge*](https://shopify.dev/tutorials/build-rails-react-app-that-uses-app-bridge-authentication) to make your development environment accessible to the internet using Ngrok.
```

Comment:
@nelsonwittwer Should we be intentionally steering people towards Cloudflare rather than Ngrok now?

Reply:
Yes, we last updated the docs to use Cloudflare instead of Ngrok because Ngrok accounts kept getting blocked from Chrome. From what I could tell OAuth redirects to Ngrok hosts looked like phishing and Google would ban accounts. Cloudfare hasn't had that behavior so we have been pointing people that direction.

Reply:
I've updated the changes to list the steps to use Cloudflared and not link to the shopify.dev docs based on the above